### ISSUE TITLE
Accessibility: MAS40B: Inspect: Name property is not descriptive for the "Snippets Aplenty" video

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#46]
#A11yMAS
#MAS40B
#Wipro
#vscode-website

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Version-10.0.14393 Build 14393.607
Browser-Edge
Edge Version-Microsoft Edge 38.14393.607.0

**Repro Steps:**
1. Open https://code.visualstudio.com/blogs in EDGE.
2. Navigate to the "Snippets Aplenty" Video
3. Observe the "Name" property in Inspect

**Actual Result:**
The "Name" property is "Video"

**Narrator Behaviour:**
Narrator announces as "Video

**Expected Result:**
The "Name" property should be descriptive.
Eg: "Snippets Aplenty Video"

**User Impact:**
Visually impaired users  will not get to know the purpose of the video as the screen reader will not announce meaningful name when the user arrives at the video
Providing meaningful name property enables the compatibility with the Screen Reader, which may be used by people with disabilities.

**Screenshots:**
![name property not descriptive for snippets aplenty video](https://cloud.githubusercontent.com/assets/24825441/21718169/98d5e9e6-d43b-11e6-86dc-71b7a531fe6c.png)

**Comment by waderyan** [comment]
Fixed with 5310dd57072354fbd8c4954c3772278353e35483
